src/api/sourceApi.ts

```diff
@@ -93,11 +94,25 @@ const getById = (id, t, params = {}) => {
         });
 };
 
+    const checkSource = (source, t) => {
+        return axios.get('/check-source', {params: {source}})
+        .then((response) => {
+            if (response.status === 404 || response.status === 500) {
```

Comment:
Suggestion: after throwing an error in the backend for invalid sources, we can simply remove this check because it will fall in the catch statement

in the catch statement we could include a simple throw, something like:

```
.catch((error) => {
  message.error(t("sources:sourceInvalid"));
  throw error
});
```